Makefile

```diff
@@ -15,6 +15,11 @@ build:
 		"-X main.Version=$(VERSION)" \
 		./cmd/telegraf/telegraf.go
 
+build-for-docker:
+	GO_ENABLED=0 GOOS=linux go build -o telegraf -ldflags \
```

Comment:
Did you mean `CGO_ENABLED=0`?

Could you also explain why this is necessary? I believe some of the linux system plugins rely on CGO and some of them may not function with this flag.